test/api/get.spec.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env mocha */
```

Comment:
I'm also getting am error from this file, which makes all the browser tests to fail:

```
ERROR in ./test/api/get.spec.js
Module build failed: TypeError: Cannot read property 'apply' of undefined
```